ajax/libs/typeboost.css/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "typeboost.css",
+  "filename": "typeboost.min.css",
+  "version": "0.1.8",
+  "description": "Typeboost.css is a set of CSS rules to ensure readable, scalable and responsive typography. No dependencies. REMs are used for font sizes, paddings, and margins. Wrap your HTML content with class .col and you are done.",
+  "homepage": "https://englishextra.github.io/libs/typeboost.css/",
+  "keywords": [
+    "typeboost.css"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/englishextra/typeboost.css.git",
+    "fileMap": [
+      {
+        "basePath": "css",
```

Comment:
>>We don't prefer to use js or css as basePath. Please leave basePath empty. Thanks.

How do you mean? Doesn't that mean that I have to change my repo structure? If it does. that would be strange. The dev decides their repo file tree. Or they don't? The basePath becomes useless as well. Please confirm do you want me to change the residence of main files from css as they are in the repo to the parent folder? Just for the purpose of basePath to keep empty?